package.json

```diff
@@ -160,7 +160,7 @@
     "prettier": "^2.2.1",
     "progress-bar-webpack-plugin": "^2.1.0",
     "prop-types": "^15.7.2",
-    "react-qr-scanner": "git+https://github.com/kybarg/react-qr-scanner.git",
+    "react-qr-scanner": "git+https://github.com/swaponline/react-qr-scanner.git",
```

Comment:
https://github.com/kybarg/react-qr-scanner/pull/41 is merged. This change is no longer needed?